### DISCUSSION TITLE
Custom survey?

### DISCUSSION THREAD

**Discussion by westjl** [Stellarium/stellarium#1789] [General]
Hello! I have been using this great piece of software for years, but I haven't looked at the new features in awhile. I was excited to discover the new HiPS survey option. I was wondering where these survey files are queried from? I haven't dug into the code but a quick look through the files suggests that these are being queried from a remote server (i.e., I can't find the files in the git repository)? I was particularly interest in adding my own custom survey that is stored locally. Is this possible?

**Comment by gzotti** [comment]
The Stellarium User Guide says yes. Section 10.2.

**Reply by westjl** [reply]
Thanks for the reply! This is awesome, unfortunately I can't get it to work.

The user guide talks about Windows and Linux, but I'm on a Mac. I can't find the file config.ini on my Mac? Any insights here?

**Reply by gzotti** [reply]
please flip a few pages to section 5. The current developers have Windows and Linux systems only, but instructions should be close enough also for Mac.

**Reply by westjl** [reply]
Yes, thanks. I had looked at that section, but I am still unable to find these configuration settings. I'll keep digging though!

**Reply by alex-w** [reply]
> Yes, thanks. I had looked at that section, but I am still unable to find these configuration settings. I'll keep digging though!

Please enable visibility of Libraries in Finder

**Reply by westjl** [reply]
Thank you! That worked and I was able to find the config file. 

I also started a local web server to host my local files and they show up in a browser. I added this line:
sources/1/url                          = http://localhost/LOCAL_P_mysurvey/properties
sources/size                           = 1

The "properties" looks to contain the same metadata info as I can see in the hipslist files (e.g. https://data.stellarium.org/surveys/hipslist). However my survey isn't showing up in the list when I run Stellarium. Is it looking for a specific keyword to use as the survey title? I thought it might be "obs_title" but that doesn't seem to be the case (I have "obs_title" defined in my properties. Any ideas why this doesn't work?

**Reply by westjl** [reply]
I got it to work. I was missing the keyword "hips_service_url" in my definition file.
